diff --git a/dist/index.d.ts b/dist/index.d.ts
index 9677cf4508e1e270fa30b606b62786fb725ca5e3..cd03ef90e8098b4b7133d3a0ce7f85c898cce06d 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -9,19 +9,19 @@ export interface StoryState {
 }
 type AnyFn = (...args: any[]) => unknown;
 type MinimalStoryObj = {
-    tags?: string[];
+    tags?: string[]|undefined;
     parameters?: {
         liveCodeEditor?: {
             disable: boolean;
             id: string;
-        };
+        }|undefined;
         docs?: {
-            source?: Record<PropertyKey, unknown>;
+            source?: Record<PropertyKey, unknown>|undefined;
             [k: string]: any;
-        };
+        }|undefined;
         [k: string]: any;
     };
-    render?: AnyFn;
+    render?: AnyFn|undefined;
     [k: string]: any;
 };
 type MinimalStory = MinimalStoryObj | (AnyFn & MinimalStoryObj);
@@ -33,10 +33,10 @@ export declare function makeLiveEditStory<T extends MinimalStory>(story: T, { co
  * React component containing a live code editor and preview.
  */
 export declare function Playground({ availableImports, code, height, id, Container, ...editorProps }: Partial<StoryState> & {
-    height?: string;
-    id?: string;
+    height?: string|undefined;
+    id?: string|undefined;
     Container?: React.ComponentType<{
         editor: React.ReactNode;
         preview: React.ReactNode;
-    }>;
+    }>|undefined;
 }): React.JSX.Element;
